packages/node/src/integrations/console.ts

```diff
@@ -0,0 +1,88 @@
+import { addBreadcrumb } from '@sentry/shim';
```

Comment:
Can we refactor this a bit, I don't really like the nested function stuff, it's hard to read and understand.